lib/node_modules/@stdlib/assert/has-to-primitive-symbol-support/test/test.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var proxyquire = require( 'proxyquire' );
+var Symbol = require( '@stdlib/symbol/ctor' );
+var detect = require( '@stdlib/assert/has-to-primitive-symbol-support/lib' );
```

Comment:
```suggestion
var detect = require( './../lib' );
```

Please make sure you have properly setup your local environment. Several of these mistakes should have been caught by local linting via Git commit hooks.